### PR TITLE
Google social login

### PR DESCRIPTION
**Pull request by marcoacierno**

### CODE REVIEW

File: frontend/src/components/login-form/index.tsx [hunk 1]
```diff
@@ -54,43 +55,81 @@ export const LoginForm: React.SFC<RouteComponentProps<{ lang: string }>> = ({
...
-        id="login-password"
-        {...password("password")}
-        required={true}
-        type="password"
-      />
-
-      <Button size="medium" palette="primary" isLoading={loading} type="submit">
-        <FormattedMessage id="login.loginButton" />
-      </Button>
+      {location && location.state && location.state.message && (
```
Comment (patrick91) [inline frontend/src/components/login-form/index.tsx]: nit: I think we can use the ? operator now: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

File: frontend/src/components/login-form/index.tsx [hunk 2]
```diff
@@ -54,43 +55,81 @@ export const LoginForm: React.SFC<RouteComponentProps<{ lang: string }>> = ({
...
+          maxWidth: "container",
+          mx: "auto",
+          mt: 3,
+          gridTemplateColumns: [null, null, "1fr 1fr"],
+          gridColumnGap: 5,
+        }}
+      >
+        <Box
+          sx={{
+            maxWidth: "smallContainer",
```
Comment (patrick91) [inline frontend/src/components/login-form/index.tsx]: do we need a small container as well? :D

Comment (marcoacierno) [reply frontend/src/components/login-form/index.tsx]: Uhm, no actually! that was before I changed the parent to be a grid, nice catch thanks

File: frontend/src/components/login-form/index.tsx (hunk 1 above)
Comment (marcoacierno) [reply frontend/src/components/login-form/index.tsx]: Changed!

File: backend/pycon/settings/base.py [hunk 3]
```diff
@@ -145,7 +145,7 @@
 SOCIAL_AUTH_POSTGRES_JSONFIELD = True
 
 SOCIAL_AUTH_LOGIN_REDIRECT_URL = f"{FRONTEND_URL}/login/success/"
-SOCIAL_AUTH_LOGIN_ERROR_URL = f"{FRONTEND_URL}/login/error/"
+# SOCIAL_AUTH_LOGIN_ERROR_URL = f"{FRONTEND_URL}/login/"
```
Comment (patrick91) [inline backend/pycon/settings/base.py]: do we need this? :)

**Pull request merged**